sources/packages/web/src/components/aest/institution/modals/ApproveProgramModal.vue

```diff
@@ -12,11 +12,18 @@
       ></formio>
     </template>
     <template v-slot:footer>
-      <footer-buttons
-        primaryLabel="Approve now"
-        @primaryClick="approveProgram"
-        @secondaryClick="dialogClosed"
-      />
+      <check-a-e-s-t-permission-role
```

Comment:
Can you please rename this component to <check-aest-permission-role> ?

Reply:
I will resolve this conversation based on further comment from @andrewsignori-aot .